<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多文档编辑器</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_newFile"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAnother"/>
    <addaction name="separator"/>
    <addaction name="action_print"/>
    <addaction name="action_printpreview"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="separator"/>
    <addaction name="action_font"/>
    <addaction name="separator"/>
    <addaction name="action_find"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(W)</string>
    </property>
    <addaction name="action_closed"/>
    <addaction name="action_closeAll"/>
    <addaction name="separator"/>
    <addaction name="action_windowLayer"/>
    <addaction name="action_tileHorizontal"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(O)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>字体</string>
     </property>
     <addaction name="action_bold"/>
     <addaction name="action_italic"/>
     <addaction name="action_underline"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="action_left"/>
     <addaction name="action_mid"/>
     <addaction name="action_right"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="menu_2"/>
    <addaction name="action_color"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_W"/>
   <addaction name="menu_O"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_file">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_newFile"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_saveAnother"/>
   <addaction name="separator"/>
   <addaction name="action_print"/>
   <addaction name="action_printpreview"/>
  </widget>
  <widget class="QToolBar" name="toolBar_window">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_closed"/>
   <addaction name="action_closeAll"/>
   <addaction name="separator"/>
   <addaction name="action_tileHorizontal"/>
   <addaction name="action_windowLayer"/>
  </widget>
  <widget class="QToolBar" name="toolBar_edit">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
   <addaction name="action_cut"/>
   <addaction name="action_copy"/>
   <addaction name="action_paste"/>
   <addaction name="separator"/>
   <addaction name="action_font"/>
   <addaction name="separator"/>
   <addaction name="action_find"/>
  </widget>
  <widget class="QToolBar" name="toolBar_o">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_bold"/>
   <addaction name="action_italic"/>
   <addaction name="action_underline"/>
   <addaction name="separator"/>
   <addaction name="action_left"/>
   <addaction name="action_mid"/>
   <addaction name="action_right"/>
   <addaction name="separator"/>
   <addaction name="action_color"/>
  </widget>
  <action name="action_newFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/新建.png</normaloff>:/icon/新建.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="toolTip">
    <string>新建一个文档</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/打开.png</normaloff>:/icon/打开.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="toolTip">
    <string>打开一个文档</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/保存.png</normaloff>:/icon/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存当前文档</string>
   </property>
  </action>
  <action name="action_saveAnother">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/另存为.png</normaloff>:/icon/另存为.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/打印.png</normaloff>:/icon/打印.png</iconset>
   </property>
   <property name="text">
    <string>打印(P)</string>
   </property>
  </action>
  <action name="action_printpreview">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/打印预览.png</normaloff>:/icon/打印预览.png</iconset>
   </property>
   <property name="text">
    <string>打印预览</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>关闭(X)</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/撤回.png</normaloff>:/icon/撤回.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/重做.png</normaloff>:/icon/重做.png</iconset>
   </property>
   <property name="text">
    <string>重做(R)</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/剪贴.png</normaloff>:/icon/剪贴.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/复制.png</normaloff>:/icon/复制.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/粘贴.png</normaloff>:/icon/粘贴.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
  </action>
  <action name="action_font">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/字体.png</normaloff>:/icon/字体.png</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="action_find">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/查找.png</normaloff>:/icon/查找.png</iconset>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="action_closed">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/关闭.png</normaloff>:/icon/关闭.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_closeAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/关闭全部.png</normaloff>:/icon/关闭全部.png</iconset>
   </property>
   <property name="text">
    <string>关闭所有</string>
   </property>
  </action>
  <action name="action_windowLayer">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/窗口层叠.png</normaloff>:/icon/窗口层叠.png</iconset>
   </property>
   <property name="text">
    <string>窗口层叠</string>
   </property>
  </action>
  <action name="action_tileHorizontal">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/水平平铺 .png</normaloff>:/icon/水平平铺 .png</iconset>
   </property>
   <property name="text">
    <string>水平平铺</string>
   </property>
  </action>
  <action name="action_color">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/颜色.png</normaloff>:/icon/颜色.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="action_bold">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/加粗.png</normaloff>:/icon/加粗.png</iconset>
   </property>
   <property name="text">
    <string>加粗(B)</string>
   </property>
  </action>
  <action name="action_italic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/倾斜.png</normaloff>:/icon/倾斜.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(I)</string>
   </property>
  </action>
  <action name="action_underline">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/下划线.png</normaloff>:/icon/下划线.png</iconset>
   </property>
   <property name="text">
    <string>下划线(U)</string>
   </property>
  </action>
  <action name="action_left">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/左对齐.png</normaloff>:/icon/左对齐.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="action_mid">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/居中.png</normaloff>:/icon/居中.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="action_right">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/右对齐.png</normaloff>:/icon/右对齐.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/关于.png</normaloff>:/icon/关于.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
